awx/ui/package.json

```diff
@@ -24,8 +24,7 @@
     "test": "karma start test/spec/karma.spec.js",
     "jshint": "grunt jshint:source --no-color",
     "test:ci": "npm run test -- --single-run --reporter junit,dots --browsers=chromeHeadless",
-    "e2e": "./test/e2e/runner.js --config ./test/e2e/nightwatch.conf.js --suiteRetries=2",
-    "e2e-no-retry": "./test/e2e/runner.js --config ./test/e2e/nightwatch.conf.js",
+    "e2e": "./test/e2e/runner.js --config ./test/e2e/nightwatch.conf.js --suiteRetries=${E2E_RETRIES}",
```

Comment:
I think the name of the env var could be prefixed with `AWX_` since it seems to be the pattern used on the others.

Reply:
ok, i'll change it